### ISSUE TITLE
SystemJS overwrites my global variable: System 

### ISSUE THREAD

**Issue by CristhianUNSa** [systemjs/systemjs#1725]
I'm having problems when using my "System" global variable. I have been working with this variable for a bunch of years so it is everywhere in the project. Now I'm migrating some code to React, but for non-breaking of my app purposes, I need this variable to stay right there.  Is there any way of SystemJS not being obstrusive with this variable (e.g. jQuery.noConflict()). I cannot rename it just beacuse what I said before. I'm using jspm with SystemJS. Thanks

**Comment by guybedford** [comment]
Would a simple fix like:

```html
<script src="system.js"></script>
<script src="yoursystemglobalthing.js"></script>
```

Or alternatively something like:

```html
<script>window.tmpSystem = System</script>
<script src="system.js"></script>
<script>window.System = tmpSystem</script>
```

**Comment by CristhianUNSa** [comment]
I think it is not a bad idea to implement a feature like this, but I don't know how many people will be favored, so I will close it. I just ended renaming all (I hope) my variable references. I tried your possible fixes before, but only worked the first call... This line `SystemJS.import('app.js')` worked beautifully, but it didn't work with this 
```
    System.import('systemjs-hot-reloader').then((connect) => {
        connect()
        System.import('app.js') //System here is not SystemJS variable
    })
```
Thanks for your response and hard work with SystemJS. I really like it
